### PR TITLE
add face_recognition from pip

### PR DESCRIPTION
**Pull request by fmessmer**
as available here: https://pypi.org/project/face_recognition/

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -2516,6 +2523,7 @@ python-pynmea2:
   ubuntu:
     pip:
       packages: [pynmea2]
+
```
Comment (mikaelarguedas) [inline rosdep/python.yaml]: please revert this change

**Review by mikaelarguedas — COMMENTED** [review]
pip rule looks good to me :+1: 
Please remove the unrelated whitespace change

**Comment by ipa-fxm** [comment]
thanks for the review...removed the blank line

**Comment by ipa-fxm** [comment]
removed gentoo

**Review by mikaelarguedas — APPROVED** [review]
lgtm, thanks @ipa-fxm for iterating :+1:

File: rosdep/python.yaml [hunk 2]
```diff
@@ -991,6 +991,19 @@ python-expiringdict:
     pip:
       packages: [expiringdict]
   ubuntu: [python-expiringdict]
+python-face-recognition:
```
Comment (dirk-thomas) [inline rosdep/python.yaml]: Please use the suffix `-pip` for pip-only entries as described in the contribution guide.

Comment (ipa-fxm) [reply rosdep/python.yaml]: done
